### PR TITLE
Added Micromouse Project

### PR DESCRIPTION
**Pull request by SurajSonawane2415**
Adding Micromouse(A small size maze solving bot) Project to official project list. 
  Please review and provide feedback. Thank you!

### CODE REVIEW

**Comment by aPR0T0** [comment]
Be more descriptive and make the project sound interesting

File: _projects/Micromouse.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+| Project Domains                                                           | Mentors                      | Project Difficulty |
+|---------------------------------------------------------------------------|------------------------------|--------------------|
+| PCB Designing, Embedded C, Control Systems, Graph Algorithms, Electronics | Suraj Sonawane, Atharva Atre | Hard               |
+
+<br>
+
+### Project Description
+
+Develop a fastest micromouse maze-solving bot based on a PID controller that can find the shortest path in a given maze.
```
Comment (aPR0T0) [inline _projects/Micromouse.md:16]: Add some images and gifs for the project and make this more interesting, use GPT!!

File: _site/projects/Micromouse/index.html [hunk 2]
```diff
@@ -0,0 +1,291 @@
+<!DOCTYPE html>
```
Comment (aPR0T0) [inline _site/projects/Micromouse/index.html:1]: Remove this file from the commits

File: _site/projects/index.html [hunk 3]
```diff
@@ -124,6 +124,20 @@ <h1 class="post-title">Eklavya Projects</h1>
 <!-- Display projects without categories --><!-- Generate cards for each project -->
```
Comment (aPR0T0) [inline _site/projects/index.html:124]: remove this file from the commit as well (basically remove the folder of _site from your commit only)

**Review by aPR0T0 — CHANGES_REQUESTED** [review]

File: _projects/Micromouse.md (hunk 1 above)
Comment (SurajSonawane2415) [reply _projects/Micromouse.md:16]: Okay I will add.

**Comment by SurajSonawane2415** [comment]
@aPR0T0 I added more project details and a GIF. Please review.

**Comment by NachtSpyder04** [comment]
> @aPR0T0 I added more project details and a GIF. Please review. image

The gif seems laggy to me, could you please share a screen recording for the same when you host it locally

**Review by NachtSpyder04 — COMMENTED** [review]
LGTM!

**Comment by SurajSonawane2415** [comment]
[Screencast from 05-14-2024 09:31:34 PM.webm](https://github.com/SRA-VJTI/eklavya.github.io/assets/129578177/3a308829-1444-4851-aab9-7f8b472e967d)

**Review by aPR0T0 — APPROVED** [review]

**Pull request merged**